tedana/metrics/kundu_fit.py

```diff
@@ -113,59 +118,53 @@ def dependence_metrics(catd, tsoc, mmix, t2s, tes, ref_img,
     totvar_norm = (WTS**2).sum()
 
     # compute Betas and means over TEs for TE-dependence analysis
-    betas = get_coeffs(catd, mmix, np.repeat(mask[:, np.newaxis], len(tes),
-                                             axis=1))
-    n_samp, n_echos, n_components = betas.shape
-    n_voxels = mask.sum()
-    n_data_voxels = (t2s != 0).sum()
+    betas = get_coeffs(utils.unmask(catd, mask),
+                       mmix,
+                       np.repeat(mask[:, np.newaxis], len(tes), axis=1))
+    betas = betas[mask, ...]
+    n_voxels, n_echos, n_components = betas.shape
     mu = catd.mean(axis=-1, dtype=float)
     tes = np.reshape(tes, (n_echos, 1))
     fmin, _, _ = getfbounds(n_echos)
 
-    # mask arrays
-    mumask = mu[t2s != 0]
-    t2smask = t2s[t2s != 0]
-    betamask = betas[t2s != 0]
-
     # set up Xmats
-    X1 = mumask.T  # Model 1
-    X2 = np.tile(tes, (1, n_data_voxels)) * mumask.T / t2smask.T  # Model 2
+    X1 = mu.T  # Model 1
+    X2 = np.tile(tes, (1, n_voxels)) * mu.T / t2s.T  # Model 2
```

Comment:
What's the motivation for the name change here (i.e., `n_data_voxels` to `n_voxels`) ? Did we change the meaning or just the name ?

Reply:
`n_data_voxels` and `n_voxels` originally corresponded to `t2s != 0` and `mask`, respectively. With the replacement of `mask` with `t2s != 0`, they became duplicates of one another.